extensions/kube-object-event-status/package.json

```diff
@@ -8,17 +8,18 @@
     "styles": []
   },
   "scripts": {
-    "build": "webpack --config webpack.config.js",
-    "dev": "npm run build --watch",
+    "build": "webpack && npm pack",
+    "dev": "webpack --watch",
     "test": "echo NO TESTS"
   },
+  "files": [
+    "dist/**/*"
+  ],
   "dependencies": {},
   "devDependencies": {
     "@k8slens/extensions": "file:../../src/extensions/npm/extensions",
     "ts-loader": "^8.0.4",
     "typescript": "^4.0.3",
-    "webpack": "^4.44.2",
```

Comment:
Removed mobx/react from package.json -> they will be resolved from top-level `node_modules` (less notifications from depandabot).